ansible/roles/apigateway/tasks/deploy.yml

```diff
@@ -28,3 +29,6 @@
   until: result.status == 200
   retries: 12
   delay: 5
+
+- name: install tzdata on apigateway
+  shell: docker exec apigateway sh -c "apk add -u tzdata && api-gateway -s reload"
```

Comment:
This is a hack, please add the package to the apigateway image in their repo here https://github.com/apache/incubator-openwhisk-apigateway/blob/master/Dockerfile#L27

then you can bump the [version of the apigateway](https://github.com/apache/incubator-openwhisk/blob/master/ansible/group_vars/all#L253) in a subsequent PR

Reply:
updated